### PR TITLE
Added python book in Ukrainian

### PR DESCRIPTION
**Pull request by ckorobov**
## What does this PR do?
Add resource(s)

### Why is this valuable (or not)?
This is translation of python documentation to Ukrainian language

### How do we know it's really free?
Original (http://docs.python.org/tut/tut.html) is free and this translation is available for free as well

### For book lists, is it a book? For course lists, is it a course? etc.
This is more like a book - description of Python language and how to use it.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-uk.md [hunk 1]
```diff
@@ -40,7 +40,7 @@
 ### Language Agnostic
 
 * [Дизайн-патерни - просто, як двері](http://designpatterns.andriybuday.com) - Андрій Будай
-
+* []
```
Comment (eshellman) [inline books/free-programming-books-uk.md:43]: typo?

Comment (ckorobov) [reply books/free-programming-books-uk.md]: yes, wanted to add another book, but it wasn't available anymore. Removed it.

**Comment by ckorobov** [comment]
@eshellman should anything else be addressed or this can be merged now?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**